docs/maatregelen/2-owp-18-maak-vereisten-onderdeel-van-subgunningscriteria.md

```diff
@@ -1,8 +1,9 @@
 ---
 # vul hier een titel in voor deze maatregel
-title: Neem de vereiste op als een subgunningscriteria bij gunningscriteria beste prijs-kwaliteitverhouding.
+title: Maak vereisten onderdeel van (sub)gunningscriteria bij gunningscriteria beste prijs-kwaliteitverhouding.
```

Comment:
Deze titel vind ik een beetje verwarrend. Klopt ie zo nog steeds?

```suggestion
title: Maak vereisten onderdeel van (sub)gunningscriteria bij een aanbesteding.
```

De specifieke toelichting volgt dan in de uitwerking van de maatregel.